cli/cmd/policy.go

```diff
@@ -0,0 +1,202 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	policyCmdState = struct {
+		File string
+		Repo bool
+		URL  string
+	}{}
+
+	// policyCmd represents the policy parent command
+	policyCmd = &cobra.Command{
+		Hidden: true,
+		Use:    "policy",
+		Short:  "manage policies",
+		Long: `Manage policies.
+
+NOTE: This feature is not yet available!`,
+	}
+
+	// policyCreateCmd represents the policy create command
+	policyCreateCmd = &cobra.Command{
+		Use:   "create",
+		Short: "create a policy",
+		Long: `Create a policy.
+
+A policy is represented in JSON format.
+The following attributes are minimally required:
+{
+    "policy_id": "lacework-example-1",
+    "title": "My Policy",
+    "enabled": false,
+    "lql_id": "MyQuery",
+    "severity": "high",
+    "description": "My Policy Description",
+    "remediation": "My Policy Remediation"
+}`,
+		Args: cobra.NoArgs,
```

Comment:
uhm, you are not passing any arguments but down in the functions you seem to be using the variables.... 🤔

Reply:
This is most likely copy/pasta issue from `inputQuery` which does use `args`.  Will fix.

Reply:
Fixed.